linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -3097,17 +3182,26 @@ private object FunctionEmitter {
     case object Discard extends Lhs
   }
 
+  sealed abstract class CaptureAction
+  object CaptureAction {
+    final case object Replace extends CaptureAction
+    final case class Fallthrough(name: LocalName) extends CaptureAction
```

Comment:
I associate "fallthrough" to a control-flow concept, such as a JS switch `case:` statement falling through to the next one when there is no `break;`. I don't really understand what is its meaning in this context.

Can we find a better name, and/or explain in doc comments what these things are?